deployment/ansible/roles/gtsite.awscli/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Install awscli
+  pip: name=awscli version={{ awscli_version }}
```

Comment:
Thanks. I knew we had this role but I couldn't find it; I was searching for `ansible-awscli`.